src/components/impactstudio/ActivityTimelineEventDetails.vue

```diff
@@ -117,7 +117,7 @@ export default {
     }
   }),
   created() {
-    const suffix = `?from=impactStudio#suitec_assetId=${this.asset.id}&suitec_userId=${this.$currentUser.id}`
+    const suffix = `?from=impactStudio#suitec_assetId=${this.asset.id}`
```

Comment:
The fix for https://jira-secure.berkeley.edu/browse/COL-2135

You may want to review this PR with `?w=1`